src/Features/Core/Portable/ConvertToInterpolatedString/AbstractConvertConcatenationToInterpolatedStringRefactoringProvider.cs

```diff
@@ -45,6 +45,14 @@ public override async Task ComputeRefactoringsAsync(CodeRefactoringContext conte
             // concats as well.  Walk to the top of that entire chain.
             var selectedExpression = token.Parent;
             var top = selectedExpression;
+
+            while (syntaxFacts.IsInterpolatedStringExpression(top) ||
```

Comment:
I am not sure the logic or reasoning behind when it get offered. can we just let users have selection when it is not the straight-forward case what to concentrate?